api/src/main/resources/notebooks.yaml

```diff
@@ -0,0 +1,362 @@
+swagger: '2.0'
+info:
+  title: Leonardo
+  description: |
+    Workbench notebooks service.
+  version: '0.1'
+  license:
+    name: BSD
+    url: 'http://opensource.org/licenses/BSD-3-Clause'
+  termsOfService: 'https://github.com/broadinstitute/leonardo'
+
+produces:
+  - application/json
+tags:
+  - name: test
+    description: Test API
+  - name: cluster
+    description: Clusters API
+  - name: notebooks
+    description: Notebooks API
+
+host: leonardo.dsde-dev.broadinstitute.org
```

Comment:
We should comment somewhere that this is only going to work inside Broad's network.

Reply:
I'll do that. Where do you think that comment should go? I was thinking probably at the top of the ClusterController? Or above the host line could also work.

Reply:
I'd put it in both places.